pkg/api-server/testdata/resources/inspect/dataplanes/_policies/merged_outbound_polices.golden.json

```diff
@@ -0,0 +1,24 @@
+{
+ "policies": [
+  {
+   "conf": [
+    {
+     "connectionTimeout": "2s",
+     "idleTimeout": "20s",
+     "http": {
+      "requestTimeout": "2s"
+     }
+    }
+   ],
+   "kind": "MeshTimeout",
+   "origins": [
+    {
+     "kri": "kri_mt_default___mt-2_"
+    },
+    {
+     "kri": "kri_mt_default___mt-1_"
+    }
+   ]
```

Comment:
Are we sure about this ordering? I would have thought it was the other way round (mt-1 first then mt-2)